elements/rh-code-block/docs/40-code.md

```diff
@@ -0,0 +1,72 @@
+{% renderInstallation %}{% endrenderInstallation %}
+
+## Usage
+
+The content of code-block snippets must be contained within a non-executable 
+`<script>` tag. By "non-executable", we mean that the script element must have
+a `type` attribute which is not `module`, `importmap` `javascript`, or any of
+the [executable javascript mimetypes][mime]. With the exception of javascript,
+use the correct mime type for your content, e.g. `text/html` for HTML content.
+
+```html
+<rh-code-block>
+  <script type="text/html">
+<!DOCTYPE html>
+<title>Title</title>
+<style>body {width: 500px;}</style>
+<body>
+  <p checked class="title" id="title">Title</p>
+    <!-- here goes the rest of the page -->
+</body>
+  </script>
+</rh-code-block>
+```
+
+{% alert state="warning", title="Warning" %}
+When slotting HTML content into the code-block, if that HTML content contains 
+a `</script>` end tag, you must escape it.
+{% endalert %}
+
+One approach to escaping script tags that is to close the containing `<script 
+type="text/html">` within the sample's closing script tag, then immediately open 
+a new sample script tag containing the rest of the snippet. Another method is to 
+insert a [zero-width-joiner][zwj] (or some other unusual unicode character) in 
+the closing script tag, and use javascript to remove it before copying the 
+content to the clipboard. Each method has benefits and drawbacks.
+
+```html
+<rh-code-block>
+  <script type="text/html">
+<p>Script tags in HTML must be escaped</p>
+<script>console.log('Success!');<</script><script type="text/html">/script>
+  </script>
+</rh-code-block>
+```
+
+To add javascript content, use the `text/sample-javascript` type.
+
+```html
+<rh-code-block>
+  <script type="text/sample-javascript">
+  import '@rhds/elements/rh-code-block/rh-code-block.js';
+  </script>
+</rh-code-block>
+```
+
+{% renderSlots %}{% endrenderSlots %}
+
+{% renderAttributes %}{% endrenderAttributes %}
+
+{% renderMethods %}{% endrenderMethods %}
+
+{% renderEvents %}{% endrenderEvents %}
+
+{% renderCssParts for='rh-code-block', level=3 %}{% endrenderCssParts %}
+
+{% renderCssCustomProperties for='rh-code-block', level=3 %}{% endrenderCssCustomProperties %}
+
+{% renderInstallation %}{% endrenderInstallation %}
```

Comment:
You have the render installation twice on this pages, once on top and once right here.

Reply:
thanks. fixed